spec/server/tweetSearch.spec.js

```diff
@@ -387,10 +392,23 @@ describe("tweetSearch", function () {
         });
     });
 
-    describe("getSpeakers", function() {
-        it("returns speakers read in from file", function() {
+    describe("speakers ", function() {
+
+        it("getSpeakers returns speakers read in from file", function() {
             expect(tweetSearcher.getSpeakers()).toEqual(speakers);
         });
+
+        it("addSpeakers calls the write to file function", function() {
+            tweetSearcher.addSpeaker("dan");
+            expect(fs.writeFile).toHaveBeenCalled();
```

Comment:
Might also be worth checking that what arguments `fs.writeFile` gets called with, to test that the speakers list is actually changing